internal/standby/standby.go

```diff
@@ -107,15 +112,27 @@ func (s *Service) runDetector(ctx context.Context) {
 func (s *Service) CheckForOutage(currentTime time.Time) {
 	outageThreshold := s.cfg.Standby.OutageThreshold
 	timeSinceLastCmd := currentTime.Sub(s.storageSvc.GetCommandTimestamp())
-	s.logger.Debug("checking", "time since last command", timeSinceLastCmd)
+	s.logger.Debug("checking", "time since last command", timeSinceLastCmd, "current mode", s.getMode(), "currentTime", currentTime)
 	if timeSinceLastCmd > outageThreshold {
 		if s.InStandbyMode() {
 			s.logger.Info("Outage detected", "config threshold", outageThreshold, "time since last command", timeSinceLastCmd)
 			s.setMode(CommandMode)
 		} else {
 			s.logger.Debug("Ongoing outage", "config threshold", outageThreshold, "time since last command", timeSinceLastCmd)
+			err := s.planHandler.TrimPlan(currentTime)
+			if err != nil {
+				s.publisher.PublishError("trimming the plan", err)
+			}
+		}
+		currentInterval, err := s.planHandler.GetCurrentInterval(currentTime)
+		if err != nil {
+			s.publisher.PublishError("getting current command", err)
+			return
+		}
+		err = s.publisher.PublishCommand(currentInterval)
+		if err != nil {
+			s.publisher.PublishError("publishing current command", err)
```

Comment:
Do you think we will be publishing back to MQTT if there is an issue?
I have a feeling we probably will want to get back to the cloud details about the outage when it happens and what things we have done during the outage.

Just open questions here, not a call for further change in current PR

Reply:
yeah I think the publisher will need to also record its activities to file, as presumably its MQTT messages aren't going to go very far during an outage. I'll add a ticket for that